python/live_manager.py

```
@@ -207,6 +242,18 @@ async def _stream_loop(self):
         print(f"[LiveManager] Stream task terminating: {self.url[:50]}")
         self.is_running = False
         self.header_ready.set()
+
+        # Send stream ended signal to all clients before terminating
+        if self.clients:
+            stream_ended_tag = self._create_stream_ended_tag()
+            for q in list(self.clients.values()):
+                try:
+                    q.put_nowait(stream_ended_tag)
+                except Exception:
+                    pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)